draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -660,10 +687,9 @@ Observing ...> Connect -> Reconnaissance --------------------> Normal
 
     <section title="RTO Expiry while using Careful Resume">
 
-
         <t> A sender that experiences a Retransmission Time Out (RTO) expiry
         ceases to use Careful Resume.
-        The sender enters the Normal Phase.</t>
+        The sender enters the Normal Phase. If using BBR, it will enter the Drain state.</t>
 
```

Comment:
See above, I think it enters safe retreat - however that is implemented?

Reply:
Damn, missed that one. Will update to say something like entering the Drain state in "safe retreat" condition.